.github/workflows/kustomize_checks.yaml

```diff
@@ -0,0 +1,26 @@
+name: Node Kustomize checker
+
+on:
+  workflow_call:
+    inputs:
+      ENV_DIRECTORY:
+        required: true
+        type: string
+
+jobs:
+  kustomize:
+    name: 'Kustomize checks for ${{ inputs.ENV_DIRECTORY }}'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout'
+        uses: actions/checkout@master
+
+    # Kustomize template to use kustomize build command on directories that contain kustomization.yaml
+      - name: 'Kustomize Build base/'
+        uses: karancode/kustomize-github-action@v1.3.0
+        with:
```

Comment:
Let's make all these inputs configurable, also I think build options input will be needed in some cases.